vignettes/pseudo-bulk-scRNAseq.Rmd

```diff
@@ -0,0 +1,173 @@
+---
+title: "Analysing single-cell RNA-sequencing Pathways using Pseudo Bulk"
+author: "Alexander Grentner"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Analysing single-cell RNA-sequencing Pathways using Pseudo Bulk}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+## Introduction
+
+The ReactomeGSA package is a client to the web-based Reactome Analysis
+System. Essentially, it performs a gene set analysis using the latest
+version of the Reactome pathway database as a backend.
+
+This vignette shows how the ReactomeGSA package can be used to perform a
+pathway analysis of cell clusters in single-cell RNA-sequencing data.
+
+### Citation
+
+To cite this package, use
+
+```         
+Alexander Grentner, Eliot Ragueneau, Chuqiao Gong, Adrian Prinz, Sabina Gansberger, Inigo Oyarzun, Henning Hermjakob, Johannes Griss, ReactomeGSA: new features to simplify public data reuse, Bioinformatics, Volume 40, Issue 6, June 2024, btae338, https://doi.org/10.1093/bioinformatics/btae338
+```
+
+```{r setup, include = FALSE}
+    knitr::opts_chunk$set(
+    collapse = TRUE,
+    comment = "#>"
+    )
+```
+
+## Installation
+
+Install the Seurat package from CRAN or github documentation from the
+satijalab/Seurat is provided by
+<https://satijalab.org/seurat/articles/install.html> The `ReactomeGSA`
+package can be directly installed from Bioconductor:
+
+```{r, eval=FALSE}
+if (!requireNamespace("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+
+if (!require(ReactomeGSA))
+  BiocManager::install("ReactomeGSA")
+
+# install the ReactomeGSA.data package for the example data
+if (!require(ReactomeGSA.data))
+  BiocManager::install("ReactomeGSA.data")
+
+
+# install Seurat from CRAN
+if(!require(Seurat))
+    install.packages('Seurat')
+
+# install to access publlic datasetsß
+if (!require(scRNAseq))
+  BiocManager::install("scRNAseq")
+
+# install Seurat from CRAN
+if (!require(eList))
```

Comment:
Comment doesn't match the package name. Also: Why do you need the eList package here? This only provides vectorized for loops